src/app/modules/feedback/feedback.util.ts

```diff
@@ -0,0 +1,63 @@
+import { StatusCodes } from 'http-status-codes'
+
+import { MapRouteError } from '../../../types'
+import { createLoggerWithLabel } from '../../config/logger'
+import { ApplicationError, DatabaseError } from '../core/core.errors'
+import * as FormErrors from '../form/form.errors'
+
+import {
+  DuplicateFeedbackSubmissionError,
+  InvalidSubmissionIdError,
+} from './feedback.error'
+
+const logger = createLoggerWithLabel(module)
+
+export const mapRouteError: MapRouteError = (
+  error: ApplicationError,
+  coreErrorMessage?: string,
```

Comment:
can we provide a default here e.g. `'Sorry, something went wrong. Please try again.'` so it can be used for unexpected cases like `DatabaseError`?